public-ui/src/app/views/ehr-view/ehr-view.component.ts

```diff
@@ -93,7 +93,6 @@ export class EhrViewComponent implements OnInit, OnDestroy {
       this.domainId = params.id;
     });
   }
-
```

Comment:
nit: I'd revert this, generally there should be newline between method definitions

Reply:
my tslint does not allow to push code with the trailing whitespace somehow.